Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -97,7 +97,8 @@ class EdgeHitProcessor: HitProcessing {
                 return
             }
 
-            let endpoint = buildEdgeEndpoint(config: edgeConfig, requestType: EdgeRequestType.interact)
+            let overwritePath = getCustomPathFromEvent(event)
+            let endpoint = buildEdgeEndpoint(config: edgeConfig, requestType: EdgeRequestType.interact, overwritePath: overwritePath)
```

Comment:
To clean this up, I'd suggest calling `getCustomPathFromEvent` in `getEdgeConfig` and store the custom request path in the `edgeConfig` dictionary. You won't need to change the `buildEdgeEndpoint` signature that way as the `config` parameter will contain the custom path.

Reply:
I still see it as a configuration value for the request but where the configuration is coming from the event data instead of the shared state. You can also update the function description to something more generic like "Extracts all the Edge configuration keys needed for this request. Configuration keys are extracted from the Configuration shared state and the given 'event' instance."